### PR TITLE
Add Neuroglancer Dataset with Authentication

### LINKED ISSUE #3898: Support adding neuroglancer dataset with authentication

### PR DESCRIPTION
**Pull request by daniel-wer**
### URL of deployed dev instance (used for testing):
- https://___.webknossos.xyz

### Steps to test:
- Add a wk-connect instance locally
- Open the "Add Dataset" view and switch to the "Add Neuroglancer Dataset" tab
  - Add a dataset which is saved on Google Cloud Storage (I'll let you know how to get credentials)
  - The dataset should be added successfully and you should be able to see data when viewing it
- Add a Neuroglancer dataset without authentication, for example: `https://neuroglancer-demo.appspot.com/#!%7B%22layers%22:%5B%7B%22source%22:%22precomputed://gs://neuroglancer-public-data/kasthuri2011/image%22%2C%22type%22:%22image%22%2C%22name%22:%22original-image%22%2C%22visible%22:false%7D%2C%7B%22source%22:%22precomputed://gs://neuroglancer-public-data/kasthuri2011/image_color_corrected%22%2C%22type%22:%22image%22%2C%22name%22:%22corrected-image%22%7D%2C%7B%22source%22:%22precomputed://gs://neuroglancer-public-data/kasthuri2011/ground_truth%22%2C%22type%22:%22segmentation%22%2C%22selectedAlpha%22:0.63%2C%22notSelectedAlpha%22:0.14%2C%22segments%22:%5B%2213%22%2C%2215%22%2C%222282%22%2C%223189%22%2C%223207%22%2C%223208%22%2C%223224%22%2C%223228%22%2C%223710%22%2C%223758%22%2C%224027%22%2C%22444%22%2C%224651%22%2C%224901%22%2C%224965%22%5D%2C%22name%22:%22ground_truth%22%7D%5D%2C%22navigation%22:%7B%22pose%22:%7B%22position%22:%7B%22voxelSize%22:%5B6%2C6%2C30%5D%2C%22voxelCoordinates%22:%5B5519.10400390625%2C8531.5380859375%2C1196.4942626953125%5D%7D%7D%2C%22zoomFactor%22:22.573112129999547%7D%2C%22perspectiveOrientation%22:%5B0.3849591314792633%2C0.6616792678833008%2C-0.6423912644386292%2C-0.0363389253616333%5D%2C%22perspectiveZoom%22:340.35867907175077%2C%22layout%22:%224panel%22%7D`

### Issues:
- fixes #3898 

------
- [x] Ready for review

### CODE REVIEW

**Comment by daniel-wer** [comment]
Thanks for your feedback, I adjusted the caption and text to explain this is optional and when it would be needed:

non-public

Regarding the error messages, yes that should be done as part of wk-connect in my opinion (otherwise wK would need to parse the error message), do you agree @jstriebel?

**Comment by jstriebel** [comment]
@philippotto What do you mean with a "hard" error? The message should not be in the response, I agree. I guess the response is a `500`?

**Comment by philippotto** [comment]
> @philippotto What do you mean with a "hard" error? The message should not be in the response, I agree. I guess the response is a `500`?

Mainly, I meant the stack trace. The wk backend usually sends a JSON which includes details about the error message (e.g., something like `[{error: "Authentication failed"}]`). However, I don't have a strong opinion about whether wk connect should do that, too.

File: docker-compose.yml [hunk 1]
```diff
@@ -201,7 +201,7 @@ services:
 
   # webKnossos-connect
   webknossos-connect:
-    image: scalableminds/webknossos-connect:master__190
+    image: scalableminds/webknossos-connect:master__205
```
Comment (jstriebel) [inline docker-compose.yml]: I just squeezed this in here, this is the upgrade to include the nicer error message.

Comment (philippotto) [reply docker-compose.yml]: Nice, thank you!

**Pull request merged**